#57 Add documentation for first 7 USD tutorials (#58)
Inbibo/shift_docs: docs/integration_resources/resources/usd_tutorials/usd_tutorial_01.md

</figure>

5. Finally, add a new *UsdStageSave* node. Connect the **stage** output plug from the *WorldSphereAdd* node to the **stage** input plug of the new node. A new filepath value can be set to save the stage to another specified location. For the current example, leave the value empty to update the file created by the *UsdStageNew* node. The board should look like the following:

<figure>
    <img src="images/usd_tutorial_01/step_05_t01.png" alt="Added the 'UsdStageSave' node to the board for saving the stage.">
    <figcaption><b>Figure 5</b>: Added the 'UsdStageSave' node to the board for saving the stage.</figcaption>
</figure>

7. Once the nodes are connected and properly set, the generated workflow can be executed to generate the USD file. The workflow will be executed and the nodes will show a blue progress bar as they are evaluated. Once the workflow has been executed, the board should look like the following:

<figure>
    <img src="images/usd_tutorial_01/step_06_t01.png" alt="Workflow executed.">
    <figcaption><b>Figure 6</b>: Workflow executed.</figcaption>
</figure>

> ### Visualizing the Stage
> 
> Shift is shipped with native plugins to visualize and inspect the stage. They can be accessed from the *Usd* top menu:
> * **USD Outliner**: Shows the user the stage hierarchy sourced from specific Usd nodes in the active board workflow.

Reviewer: Apply to other pages
```suggestion
### Visualising the Stage

Shift is shipped with native plugins to visualise and inspect the stage. They can be accessed from the *USD* top menu:
* **USD Outliner**: Shows the user the stage hierarchy sourced from specific Usd nodes in the active board workflow.
```

Author: Done

> * **USD Viewer**: Shows the stage in the Usd viewer sourced from specific Usd nodes in the active board workflow.

Author: Done

<figure>
    <img src="images/usd_tutorial_01/step_08_t01.png" alt="Shift USD Outliner and USD Viewer plugins.">
    <figcaption><b>Figure 7</b>: Shift USD Outliner and USD Viewer plugins.</figcaption>
</figure>

> The *USD Viewer* tool shipped with USD is used to visualise and inspect the stage viewport:

Author: Question here, visualize is mostly used in American English, but visualise is predominantly used in British English. In this case, we should use visualise, right?

> The *USD Viewer* tool shipped with USD is used to visualise and inspect the stage viewport:

Reviewer: Both are correct, but we can use visualise yes. In that case we should change it in lines 68 and 70. I think what it's important is that it is consistent across the documentation.

## Creating Your First USD Stage

> This tutorial is focused on creating, using the operators from Shift_USD catalog, a simple USD stage containing a transform and a sphere.

Reviewer: ```suggestion
This tutorial is focused on creating a simple USD stage containing a transform and a sphere using the operators from Shift_USD catalog.
```

> In order to follow easily the steps, it is recommended to read the documentation about the [Shift's interface](../../../getting_started/basics/ui_overview.md) and [workflow execution](../../../getting_started/basics/execute.md).

Reviewer: I don't think this is needed.

Author: Okay, we can remove it 👍

### Tutorial

> 1. Open Shift application. A new window's instance of Shift should appear like the following one:

Reviewer: I would mention somewhere to check for the USD operators in the Node list . If not, redirect the user to the Shift_USD installation page.

Author: Okay, let me know your opinion on the change and I can copy it to the rest of the tutorials 👍

<figure>
    <img src="images/usd_tutorial_01/step_01_t01.png" alt="Shift UI.">
    <figcaption><b>Figure 1</b>: Shift UI.</figcaption>
</figure>

> 2. Now, add a new **<tt>UsdStageNew</tt>** node to the board. Using the **Inspector** widget, set as value for the **filepath** input plug a filepath pointing to a `.usda` file. For doing this, the UI offers a browse button next to the plug input field that opens a browse dialog to facilitate locating the file or it can be added typing manually the path. For the current example, the filepath can be: `D:/USD_TUTORIALS/01/HelloWorld.usda`. The board should look like the following one:

Reviewer: We don't need to be as detailed about how to set values. 
Also, will file paths be relative to the shift folder? 

```suggestion
2. Now, add a new **<tt>UsdStageNew</tt>** node to the board. Using the **Inspector** widget, set as the value for the **filepath** input plug a filepath pointing to a `.usda` file. For the current example, the filepath can be: `D:/USD_TUTORIALS/01/HelloWorld.usda`. The board should look like the following:
```

Author: Done. I was working with absolute paths just to avoid confusing the user, because the files are reused in the tutorials and in this way they can see that is the file related to the current tutorial 🤔

<figure>
    <img src="images/usd_tutorial_01/step_02_t01.png" alt="Added the 'UsdStageNew' node to the board.">
    <figcaption><b>Figure 2</b>: Added the 'UsdStageNew' node to the board.</figcaption>
</figure>

> 3. Add a new **<tt>UsdPrimAdd</tt>** node to the board. Now, connect the **stage** output plug from the `UsdStageNew` node to the **stage** input plug of the new node. Also, set `/hello` as value for the **primpath** input plug and choose `Xform` as value for the **primType** input plug. Rename the node as `HelloXformAdd` to identify it easily. The board should look like the following one:

Reviewer: ```suggestion
3. Add a new **<tt>UsdPrimAdd</tt>** node to the board. Now, connect the **stage** output plug from the `UsdStageNew` node to the **stage** input plug of the new node. Also, set `/hello` as the value for the **primpath** input plug and choose `Xform` as the value for the **primType** input plug. Rename the node as `HelloXformAdd` to identify it easily. The board should look like the following:
```

Author: Done

</figure>

5. Finally, add a new **<tt>UsdStageSave</tt>** node. Connect the **stage** output plug from the `WorldSphereAdd` node to the **stage** input plug of the new node. A new filepath value can be set to save the file to another specified file. For the current example, leave the value empty to update the file created by the `UsdStageNew` node. The board should look like the following one:

<figure>
    <img src="images/usd_tutorial_01/step_05_t01.png" alt="Added the 'UsdStageSave' node to the board for saving the stage.">
    <figcaption><b>Figure 5</b>: Added the 'UsdStageSave' node to the board for saving the stage.</figcaption>
</figure>

7. Once the nodes are connected and properly set, the generated workflow can be executed to generate the Usd file. The workflow will be executed and the nodes will show a blue progress bar as they are done. The board should look like the following one:

<figure>
    <img src="images/usd_tutorial_01/step_06_t01.png" alt="Workflow executed.">
    <figcaption><b>Figure 6</b>: Workflow executed.</figcaption>
</figure>

### Visualizing the Stage

> Use the <tt>usdview</tt> tool shipped with USD to visualise and inspect the stage:

Reviewer: Maybe we can suggest using our plugin?

Author: Yes! It can be added also to this visualizing section. I was waiting until we merged the development. Thanks for the reminder!

## Creating Your First USD Stage

This tutorial is focused on creating a simple USD stage containing a transform and a sphere using the operators from Shift_USD catalog.

### Tutorial

1. Open Shift application. A new window's instance of Shift should appear like the following one:

<figure>
    <img src="images/usd_tutorial_01/step_01_t01.png" alt="Shift UI.">
    <figcaption><b>Figure 1</b>: Shift UI.</figcaption>
</figure>

>[!NOTE]

> > Check if the USD nodes appear in the Node List widget, it can be easily confirmed if in the catalog combobox appear `sUSD` or `sUSDTypes` catalogs. If no USD nodes appear, check again the [Shift_USD installation page](../usd.md#installation).

Reviewer: We are going to leave it as it is.

> 2. Now, add a new **<tt>UsdStageNew</tt>** node to the board. Using the **Inspector** widget, set as the value for the **filepath** input plug a filepath pointing to a `.usda` file. For the current example, the filepath can be: `D:/USD_TUTORIALS/01/HelloWorld.usda`. The board should look like the following:

Reviewer 2: Why are you using `<tt>`? This has been deprecated in html

Author: Oh, yes, I forgot it, you're right.
I have 2 proposals for replacing the `<tt>` tag:

### Using the `<code>` tag:
![Code_Tag](https://github.com/user-attachments/assets/607457cd-a8fb-4ebe-b2f8-7ebb4e8a14b3)

### Using bold-italic font:
![Bold_italic](https://github.com/user-attachments/assets/981d76df-942c-4118-bc7c-73aa20657691)

Please @Inbibo/shift , let me know which is the best way for use it when naming the node/operators, referring the name of the node to be searched in the Node List widget for filtering.

Author: Okay, I changed it for the tutorial 01 and has this appearance:
![image](https://github.com/user-attachments/assets/83f49ce8-eccf-4386-a838-79ca7f6012d4)

Let me know what you do think adding the reaction: 👍  or 👎

## Creating a simple USD Stage

This tutorial is focused on creating a simple USD stage containing a transform and a sphere using the operators from Shift_USD catalog.

### Tutorial

1. Open Shift application. A new window's instance of Shift should appear like the following one:

<figure>
    <img src="images/usd_tutorial_01/step_01_t01.png" alt="Shift UI.">
    <figcaption><b>Figure 1</b>: Shift UI.</figcaption>
</figure>

>[!NOTE]

> > Before creating the USD workflow, ensure that the USD nodes appear in the Node List widget. The catalogs combobox should show `sUSD` and `sUSDTypes` if USD is correctly loaded in Shift. If this is not the case, please check again the [Shift_USD installation page](../usd#installation).

Reviewer 3: The catalog was decided to be included, resolving this

    <figcaption><b>Figure 5</b>: Added the 'UsdStageSave' node to the board for saving the stage.</figcaption>
</figure>

7. Once the nodes are connected and properly set, the generated workflow can be executed to generate the USD file. The workflow will be executed and the nodes will show a blue progress bar as they are evaluated. Once the workflow has been executed, the board should look like the following:

<figure>
    <img src="images/usd_tutorial_01/step_06_t01.png" alt="Workflow executed.">
    <figcaption><b>Figure 6</b>: Workflow executed.</figcaption>
</figure>

### Visualizing the Stage

Shift is shipped with native plugins to visualize and inspect the stage. They can be accessed from the *Usd* top menu:
* **USD Outliner**: Shows the user the stage hierarchy sourced from specific Usd nodes in the active board workflow.

* **USD Viewer**: Shows the stage in the Usd viewer sourced from specific Usd nodes in the active board workflow.

<figure>
    <img src="images/usd_tutorial_01/step_08_t01.png" alt="Shift USD Outliner and USD Viewer plugins.">
    <figcaption><b>Figure 7</b>: Shift USD Outliner and USD Viewer plugins.</figcaption>
</figure>

> Also, the *usdview* tool shipped with USD is used to visualise and inspect the stage:

Reviewer 3: ```suggestion
The *USD Viewer* tool shipped with USD is used to visualise and inspect the stage viewport:
```
